### PR TITLE
Enable transaction shuffling in release.yaml for v1.7 release

### PR DESCRIPTION
**Pull request by sitalkedia**
### Description

As above

### CODE REVIEW

**Review by danielxiangzl — APPROVED** [review]

File: aptos-move/aptos-release-builder/data/release.yaml [hunk 1]
```diff
@@ -23,7 +23,21 @@ proposals:
...
           transaction_shuffler_type:
             deprecated_sender_aware_v1: 32
           block_gas_limit : 35000
+          transaction_deduper_type: txn_hash_and_authenticator_v1
+  - name: enable_transaction_shuffling
+    metadata:
+      title: "Enable Transaction Shuffling"
+      description : "AIP-27: Sender Aware Transaction Shuffling"
+      discussion_url: "https://github.com/aptos-foundation/AIPs/issues/109"
+    execution_mode: MultiStep
```
Comment (msmouse) [inline aptos-move/aptos-release-builder/data/release.yaml:36]: why is it a multi- step when there's only one step?

Comment (sitalkedia) [reply aptos-move/aptos-release-builder/data/release.yaml:36]: I don't understand what this is, just copied the gas limit one 😂

Comment (sitalkedia) [reply aptos-move/aptos-release-builder/data/release.yaml]: Changed this to `SingleStep`

File: aptos-move/aptos-release-builder/data/release.yaml [hunk 2]
```diff
@@ -20,10 +20,24 @@ proposals:
       title: "Enable Block Gas Limit"
       description: "AIP-33: The block gas limit is a feature that terminates block execution when the gas consumed by the committed prefix of transactions exceeds the limit, thus providing predictable latency."
       discussion_url: "https://github.com/aptos-foundation/AIPs/issues/132"
-    execution_mode: MultiStep
+    execution_mode: SingleStep
```
Comment (movekevin) [inline aptos-move/aptos-release-builder/data/release.yaml:23]: Keep it as multistep please

Comment (sitalkedia) [reply aptos-move/aptos-release-builder/data/release.yaml:23]: Is this causing the test failure or that's unrelated?

Comment (sitalkedia) [reply aptos-move/aptos-release-builder/data/release.yaml:23]: Also, Can you explain what's the difference between two?

Comment (sherry-x) [reply aptos-move/aptos-release-builder/data/release.yaml]: SingleStep is executing one change, while MultiStep chain several updates together when proposing

Comment (sherry-x) [reply aptos-move/aptos-release-builder/data/release.yaml]: @movekevin why this one need to be multistep? it's just changing one config?

Comment (movekevin) [reply aptos-move/aptos-release-builder/data/release.yaml]: Single step triggers a different code path in the cli and smart contract that should now be deprecated. Cc @runtian-zhou

Comment (sherry-x) [reply aptos-move/aptos-release-builder/data/release.yaml]: Interesting, in this case, shall we rename the the flow when deprecating? or just make it the default?

Comment (movekevin) [reply aptos-move/aptos-release-builder/data/release.yaml]: Yes that's the plan. We have that listed for a while now - kill single step completely in the CLI flow. cc @thepomeranian

**Review by runtian-zhou — APPROVED** [review]

**Pull request merged**